Refactor method to explicit helper. Fixes #253

peterblazejewicz opened the pull request:
This PR refactors method creating the User model instance in controller to silence compiler warnings (as per the issue description):
- change method name to be more explicit about method role
- update method use in controller

With this change running:

```
kpm build
```

in `HelloMvc` sample project will output no warnings:

```
Build succeeded.
    0 Warnings(s)
    0 Error(s)

Time elapsed 00:00:02.4029731
```

I've already signed Microsoft CLA.
Thanks!

Commit:
Updating dev with latest on master

Tratcher committed:
Clean up samples.

Tratcher committed:
Add a solution and kproj files.

Commit:
Updating Kvm in home/dev

Eilon commented:
CLA confirmed. (Sorry for the delay, was in Hawaii :palm_tree: :palm_tree: :palm_tree:)

Praburaj commented:
Thanks Eilon. 

Sorry @peterblazejewicz just noticed your change to `master` branch. Could you send your change to `dev` branch instead?

Commit:
:arrow_up: kvm.ps1, kvm.cmd

Source: git@github.com:aspnet/Home commit 9af8d52c9fd9779827d84a8e11dde5b0d5478165

peterblazejewicz committed:
Refactor method to explicit helper. Fixes #253

- change method name to be more explicit about method role
- update method use in controller

#253

peterblazejewicz commented:
I want to close this PR in favor of this with better history:
https://github.com/aspnet/Home/pull/268
Thanks!
